api/observability/v1/output_types.go

```diff
@@ -1030,7 +1021,6 @@ type Syslog struct {
 	//     Emergency Alert Critical Error Warning Notice Informational Debug
 	//
 	// +kubebuilder:validation:Optional
-	// +kubebuilder:default:=informational
```

Comment:
If we are removing this optional default then we need to pair it with an intializer (e.g. internal/api/initialize) or something else to provide a default in the generator